Content.Server/Animals/Components/EggLayerComponent.cs

```diff
@@ -9,44 +10,47 @@ namespace Content.Server.Animals.Components;
 ///     It also grants an action to players who are controlling these entities, allowing them to do it manually.
 /// </summary>
 
-[RegisterComponent]
+[RegisterComponent, Access(typeof(EggLayerSystem)), AutoGenerateComponentPause]
 public sealed partial class EggLayerComponent : Component
 {
-    [DataField]
-    public EntProtoId EggLayAction = "ActionAnimalLayEgg";
+    /// <summary>
+    ///     The item that gets laid/spawned, retrieved from animal prototype.
+    /// </summary>
+    [DataField(required: true)]
+    public List<EntitySpawnEntry> EggSpawn = default!;
 
     /// <summary>
-    ///     The amount of nutrient consumed on update.
+    ///     Player action.
     /// </summary>
-    [DataField, ViewVariables(VVAccess.ReadWrite)]
-    public float HungerUsage = 60f;
+    [DataField]
+    public EntProtoId EggLayAction = "ActionAnimalLayEgg";
+
+    [DataField]
+    public SoundSpecifier EggLaySound = new SoundPathSpecifier("/Audio/Effects/pop.ogg");
 
     /// <summary>
     ///     Minimum cooldown used for the automatic egg laying.
     /// </summary>
-    [DataField, ViewVariables(VVAccess.ReadWrite)]
-    public float EggLayCooldownMin = 60f;
+    [DataField]
+    public float EggLayCooldownMin = 45f;
 
     /// <summary>
     ///     Maximum cooldown used for the automatic egg laying.
     /// </summary>
-    [DataField, ViewVariables(VVAccess.ReadWrite)]
-    public float EggLayCooldownMax = 120f;
+    [DataField]
+    public float EggLayCooldownMax = 90f;
```

Comment:
I'm not a fan of making large balance changes inside of bugfix PRs for 2 reasons:
1. If we need to revert the change it makes it much nastier.
2. It's much more difficult for forks to pull 1 without the other as they need to manually fix it themselves.

Reply:
Yeah, I realize it's against the guidelines.... it did start out as a tweak and while working on it, I realized there is a bug.

Should I remove those balance changes from here and make it a separate PR?

For reference, the cooldown used to be between 60s and 120s, now it would be 45s and 90s.
The more impactful balance change is the nutrient cost, which was 60 for one egg, now 25. Compared to milk which is 10 nutrient for 25u.